### PR TITLE
CTCTRADERS-2494 Log outgoing message header information at info level

### PR DESCRIPTION
**Pull request by amaalali**

### CODE REVIEW

**Review by DavidGregory084 — APPROVED** [review]
Looks great to me 👍

File: conf/application-json-logger.xml [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
     <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
 
+    <logger name="application" level="${logger.application:-WARN}"/>
```
Comment (DavidGregory084) [inline conf/application-json-logger.xml:10]: I think this `application` logger refers to the deprecated uppercase L `play.api.Logger` - I think that by changing to use the `play.api.Logging` logger, the logging above is configurable from the root logger now instead

I could be wrong though!

Comment (amaalali) [reply conf/application-json-logger.xml:10]: This is from the logger that is set by [the trait here](https://github.com/hmrc/transit-movements-trader-eis-router/blob/master/app/logging/Logging.scala#L23)

Comment (DavidGregory084) [reply conf/application-json-logger.xml:10]: Ah I see, good spot, I didn't realise it was using a custom trait rather than the Play one

**Pull request merged**